test/SlowTests/Server/Documents/ETL/ConnectionStringTests.cs

```diff
@@ -276,5 +277,25 @@ public void CanGetConnectionStringByName()
                 Assert.True(resultElastic.ElasticSearchConnectionStrings.Count > 0);
             }
         }
+
+        [RequiresMsSqlFact]
```

Comment:
Cannot it be just a regular `[Fact]`?

Reply:
sure, didn't think twice about it, my bad :)